client/my-sites/site-settings/comment-display-settings/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+
+const CommentDisplaySettings = ( { translate, onChangeField, submittingForm, fields } ) => {
+	return (
+		<FormFieldset className="comment-display-settings">
+			<FormLabel htmlFor="highlander_comment_form_prompt">{ translate( 'Comments Label' ) }</FormLabel>
+			<FormTextInput
+				name="highlander_comment_form_prompt"
+				type="text"
+				id="highlander_comment_form_prompt"
+				value={ fields.highlander_comment_form_prompt || '' }
+				onChange={ onChangeField( 'highlander_comment_form_prompt' ) }
+				disabled={ submittingForm } />
+			<FormSettingExplanation>
```

Comment:
I'd leave some margin below the explanation and the next field - currently they're a little too tight:

![](https://cldup.com/W9B0U54bgk.png)

A design opinion would be helpful here - @MichaelArestad what do you think?